cmd/eval.go

```diff
@@ -45,13 +51,15 @@ const (
 	evalJSONOutput     = "json"
 	evalValuesOutput   = "values"
 	evalBindingsOutput = "bindings"
+	evalProfileOutput  = "profile"
 )
 
 type evalResult struct {
-	Result      rego.ResultSet         `json:"result,omitempty"`
-	Explanation []string               `json:"explanation,omitempty"`
-	Metrics     map[string]interface{} `json:"metrics,omitempty"`
-	Profile     profiler.Report        `json:"profile,omitempty"`
+	Result            rego.ResultSet         `json:"result,omitempty"`
+	Explanation       []string               `json:"explanation,omitempty"`
+	Metrics           map[string]interface{} `json:"metrics,omitempty"`
+	Profile           profiler.Report        `json:"profile,omitempty"`
```

Comment:
As is, the `profile` key will show in all results since the `profiler.Report` will never be a zero value. Make it a `*profiler.Report` to fix this.